WarOOP/Models/Vampire.cs

```diff
@@ -0,0 +1,23 @@
+namespace WarOOP.Models;
+
+public class Vampire : Warrior
+{
+    public readonly int Vampirism; 
```

Comment:
Don't make fields public. This field is used only for internal logic, so you don't need to expose it at all.

Reply:
I need it for tests, but i can add private setter